packages/web-app-mail/src/components/MailList.vue

```diff
@@ -14,6 +14,19 @@
         <h2 class="text-lg ml-4" v-text="mailbox.name"></h2>
         <div class="paceholder" />
       </div>
+      <div class="mt-2 px-6">
+        <oc-button
+          id="new-email-menu-btn"
+          class="w-full"
+          v-oc-tooltip="$gettext('Create new email')"
```

Comment:
If the button contains the full string, we don’t need a tooltip or an aria-label as it’s self describing